roles/build_containers/molecule/hotfix/converge.yml

```diff
@@ -35,6 +35,7 @@
       ansible.builtin.file:
         path: /tmp/rpms
         state: directory
+        mode: "0775"
```

Comment:
some directories are marked 0755 while others are marked 0775 

is that intentional?

Reply:
this was updated so now we only have 0755
marking this one resolved thanks